modules/reference/pages/internal-metrics-reference.adoc

```diff
@@ -145,6 +191,24 @@ Number of compacted segments.
 
 ---
 
+=== vectorized_storage_log_compaction_removed_bytes
+
+Number of bytes removed from local storage by compaction operations. This is tracked on a per-topic and per-partition basis. It primarily exists to track the fact that compaction is performing operations on local storage.
```

Comment:
```suggestion
Number of bytes removed from local storage by compaction operations. This is tracked on a per-topic and per-partition basis. It tracks whether or not compaction is performing operations on cloud storage.
```

Reply:
Partially accepted - this is referring to local storage, not cloud storage. There's a separate metric that applies to cloud storage.